tensortrade/env/default/renderers.py

```diff
@@ -291,7 +288,7 @@ class PlotlyTradingChart(BaseRenderer):
     height : int
         Chart height in pixels. Affects both display and saved file
         charts. Set to None for 100% height. Default is None.
-    save_format : str
+    save_format : float
```

Comment:
Why is the type changed to float?

Reply:
Accident. Reverted this file back to what it should be.